lib/RoadizCoreBundle/src/Entity/NodeTypeField.php

```diff
@@ -4,45 +4,23 @@
 
 namespace RZ\Roadiz\CoreBundle\Entity;
 
-use Doctrine\DBAL\Types\Types;
-use Doctrine\ORM\Mapping as ORM;
 use RZ\Roadiz\Contracts\NodeType\NodeTypeFieldInterface;
 use RZ\Roadiz\Contracts\NodeType\NodeTypeInterface;
 use RZ\Roadiz\Contracts\NodeType\SerializableInterface;
 use RZ\Roadiz\Core\AbstractEntities\AbstractField;
 use RZ\Roadiz\CoreBundle\Enum\FieldType;
 use RZ\Roadiz\CoreBundle\Form\Constraint as RoadizAssert;
-use RZ\Roadiz\CoreBundle\Repository\NodeTypeFieldRepository;
-use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Serializer\Annotation as Serializer;
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
  * NodeTypeField entities are used to create NodeTypes with
  * custom data structure.
  */
-#[
-    ORM\Entity(repositoryClass: NodeTypeFieldRepository::class),
-    ORM\Table(name: 'node_type_fields'),
-    ORM\Index(columns: ['visible']),
-    ORM\Index(columns: ['indexed']),
-    ORM\Index(columns: ['position']),
-    ORM\Index(columns: ['group_name']),
-    ORM\Index(columns: ['group_name_canonical']),
-    ORM\Index(columns: ['type']),
-    ORM\Index(columns: ['name'], name: 'ntf_name'),
-    ORM\Index(columns: ['universal']),
-    ORM\Index(columns: ['node_type_id', 'position'], name: 'ntf_type_position'),
-    ORM\UniqueConstraint(columns: ['name', 'node_type_id']),
-    ORM\HasLifecycleCallbacks,
-    UniqueEntity(fields: ['name', 'nodeType']),
-    RoadizAssert\NodeTypeField
```

Comment:
Are you sure `RoadizAssert\NodeTypeField` is related to Doctrine?

I believe we still need this.